test/e2e/network/service.go

```diff
@@ -1179,37 +1179,6 @@ var _ = SIGDescribe("Services", func() {
 		framework.ExpectNoError(err)
 	})
 
-	/*
-		Create a ClusterIP service with an External IP that is not assigned to an interface.
-		The IP ranges here are reserved for documentation according to
-		[RFC 5737](https://tools.ietf.org/html/rfc5737) Section 3 and should not be used by any host.
-	*/
-	ginkgo.It("should be possible to connect to a service via ExternalIP when the external IP is not assigned to a node", func() {
```

Comment:
So the problem is that externalIP is **not** allowed for any pods inside a bridge but because some CNIs don't use bridges it was allowed in some clusters?

Reply:
So basically any cluster that is using a bridge is failing this test (as expected)?

Reply:
exactly, https://github.com/kubernetes/kubernetes/issues/95785#issuecomment-722255763

when using a ptp interface it considers that traffic is coming from outside and it goes through.

We have to sort out first the definition of ExternalIP first, this is breaking multiple CIs now ...

Reply:
> So basically any cluster that is using a bridge is failing this test (as expected)?

we should clarify that IMHO, the definition can not be done by the implementation constraints